testsuite/features/srv_spacewalk_remove_channel_tool_delete_channel.feature

```diff
@@ -0,0 +1,46 @@
+# COPYRIGHT (c) 2019 SUSE LLC
+# Licensed under the terms of the MIT License.
+
+Feature: Test delete channels with clone using spacewalk-remove-channel tool
+  we cannot delete a channel if it  have a clone
+
+  Scenario: Clone the first channel
+    Given I am on the manage software channels page
+    When I follow "Clone Channel"
+    And I select "Test-Channel-x86_64" as the origin channel
+    And I click on "Clone Channel"
+    Then I should see a "Create Software Channel" text
+    And I should see a "Current state of the channel" text
+    When I click on "Clone Channel"
+    Then I should see a "Clone of Test-Channel-x86_64" text
+
+  Scenario: Clone a second channel using first channel as base
+    Given I am on the manage software channels page
+    When I follow "Clone Channel"
+    And I select "Clone of Test-Channel-x86_64" as the origin channel
+    And I click on "Clone Channel"
+    Then I should see a "Create Software Channel" text
+    And I should see a "Current state of the channel" text
+    When I click on "Clone Channel"
+    Then I should see a "Clone of Clone of Test-Channel-x86_64" text
+
+  Scenario: Verify if both clone channels exists
+    When I list channels with spacewalk-remove-channel
+    Then I should see "clone-test-channel-x86_64" in the output
+    And I should see "clone-clone-test-channel-x86_64" in the output
+
+  Scenario: Delete channel with one clone
+    When I delete this channels with spacewalk-remove-channel:
+      |clone-test-channel-x86_64|
+    Then I should see "Error: cannot remove channel" in the output
+    And  I should see "clone channel(s) exist" in the output
+    And  I should see "clone-test-channel-x86_64" in the output
+    And  I should see "clone-clone-test-channel-x86_64" in the output
+
+  Scenario: Delete base channel and clone
+    When I delete this channels with spacewalk-remove-channel:
```

Comment:
"this channels" => "these channels"

Honestly, using a generic step for two examples is a bit too much, one step with two arguments in string would have worked as well.

Reply:
I implement the step in a generic way so it can be used in future tests of the spacewalk-remove-channel tool. I don't know in the future how many channels a step may want to delete at one row, so I let it generic enough to handle different scenarios in the future.